README.md

```diff
@@ -28,3 +28,161 @@ mvn clean install -Ptestresources -Ptest-native -Dlibnd4j.cuda=x.x -Dlibnd4j.com
 We have separate repository with various examples available: https://github.com/deeplearning4j/dl4j-examples
 
 In the examples repo, you'll also find a tutorial series in Zeppelin: https://github.com/deeplearning4j/dl4j-examples/tree/master/tutorials
+
+## Detailed Instructions to build DL4J
```

Comment:
Could the detailed instructions specific to DL4J perhaps live in the [nd4j/README.md](https://github.com/deeplearning4j/deeplearning4j/blob/master/nd4j/README.md) file and the top-level README here would sign-post the reader to there with a link of some sort?

Reply:
It's a good idea. Whatever apache developers decide :)